account_edi_simple_pdf/README.rst

```diff
@@ -0,0 +1,212 @@
+
+Install pdftotext python lib
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To install **pdftotext python lib**, run:
+
+.. code::
+
+  sudo apt install build-essential libpoppler-cpp-dev pkg-config python3-dev
+
+and then install the lib via pip:
+
+.. code::
+
+  pip3 install --upgrade pdftotext
+
+On OSes other than Debian/Ubuntu, follow the instructions on the `project page <https://github.com/jalan/pdftotext>`_.
+
+Install pdftotext command line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To install **pdftotext command line**, run:
+
+.. code::
+
+  sudo apt install poppler-utils
+
+Install pypdf
+~~~~~~~~~~~~~
+
+To install the **pypdf** python lib, run:
+
+.. code::
+
+  pip3 install --upgrade pypdf
+
+
+Other requirements
+~~~~~~~~~~~~~~~~~~
+
+This module also requires the following Python libraries:
+
+* `regex <https://pypi.org/project/regex/>`_ which is backward-compatible with the *re* module of the Python standard library, but has additional functionalities.
+* `dateparser <https://github.com/scrapinghub/dateparser>`_ which is a powerful date parsing library.
+
+The dateparser lib depends itself on regex. So you can install these Python libraries via pip with the following command:
+
+.. code::
+
+  pip3 install --upgrade dateparser
+
+The dateparser lib is not compatible with all regex lib versions. As of February 2024, the `version requirement <https://github.com/scrapinghub/dateparser/blob/master/setup.py#L36>`_ declared by dateparser for regex is **!=2019.02.19, !=2021.8.27**. So the latest version of dateparser is currenly compatible with the latest version of regex. To know the version of regex installed in your environment, run:
+
+
+.. code::
+
+  pip3 show regex
+
+Configuration
+=============
+
+By default, for the PDF to text conversion, the module tries the different methods in the order mentioned in the INSTALL section: it will first try to use **PyMuPDF**; if it fails (for example because the lib is not properly installed), then it will try to use the **pdftotext python lib**, if that one also fails, it will try to use **pdftotext command line** and, if it also fails, it will eventually try **pypdf**. If none of the 4 methods work, Odoo will display an error message.
+
+If you want to force Odoo to use a specific text extraction method, go to the menu *Configuration > Technical > Parameters > System Parameters* and create a new System Parameter:
+
+* *Key*: **invoice_import_simple_pdf.pdf2txt**
+* *Value*: select the proper value for the method you want to use:
+
+  1. pymupdf
+  #. pdftotext.lib
+  #. pdftotext.cmd
+  #. pypdf
+
+In this configuration, Odoo will only use the selected text extraction method and, if it fails, it will display an error message.
+
+You will find a full demonstration about how to configure each Vendor and import the PDF invoices in this `screencast <https://www.youtube.com/watch?v=edsEuXVyEYE>`_.
+
+Bug Tracker
```

Comment:
I suggest a short section like this:

```
Usage
=====

Go to Invoicing, press the button Upload and upload a PDF file. Now the PDF file will be processed by account_edi_simple_pdf. In case your PDF file contains Factur-X data and you have Factur-X activated, that functionality will be executed instead.
```